modules/infrastructure-node-sizing.adoc

```diff
@@ -11,19 +11,21 @@ _Infrastructure nodes_ are nodes that are labeled to run pieces of the {product-
 |===
 | Number of worker nodes |CPU cores |Memory (GB)
```

Comment:
The first two entries in this table include 500 and 1000, I'm assuming this in regards to the cluster density in the 4.12+ table. This table only has three columns. 

Also, the last two entries do not have the 4000 cluster density in the 4.12+ tables. So we have two entries with four columns and two entries with three columns. Did you want this table to look like 4.12+? 

```[options="header",cols="4*"]
|===
| Number of worker nodes |Cluster density, or number of namespaces |CPU cores |Memory (GB)

| 27
| 500
| 4
| 24

| 120
| 1000
| 8
| 48

| 252
| 4000
| 16
| 128

| 501
| 4000
| 32
| 128

|===
```